central/graphql/resolvers/test_utils.go

```diff
@@ -100,6 +106,7 @@ func testImages() []*storage.Image {
 								SetFixedBy: &storage.EmbeddedVulnerability_FixedBy{
 									FixedBy: "1.1",
 								},
+								Severity: storage.VulnerabilitySeverity_CRITICAL_VULNERABILITY_SEVERITY,
```

Comment:
The sample data was incorrect. You cannot have multiple severities for same operating system.